nncf/tensorflow/algorithm_selector.py

```diff
@@ -10,10 +10,11 @@
  See the License for the specific language governing permissions and
  limitations under the License.
 """
+from typing import Dict
+from typing import Type
```

Comment:
We should discuss and choose one way to import from typing

Reply:
See comment from above.